src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
 :relative-path: ../../../
-include::{docdir}/variables.adoc[]
\ No newline at end of file
+include::{docdir}/variables.adoc[]
+
+In this part we will explain how to use the **Standard User Workflow** in the nominal case, i.e. using the default implementations of each parts of the workflow (For example, the default implementation to send the confirmation email use the _Mailjet_ API).
+
+Use the Standard User Workflow can be done in 3 parts :
+
+* Use the Standard User Workflow in your worker
+* Add properties to configure the Standard User Workflow
+* Create the client part
+
+== Bootstrap
+
+In this Quickstart, we will begin with a generated application using the CLI. So you can run :
+
+[source, console]
+----
+$ npm init @zetapush myApp
+----
+
+We also use the package `@zetapush/user-management` so you can install it with :
+
+[source, console]
+----
+$ npm install --save @zetapush/user-management
+----
+
+== Worker part
+
+Your worker contains just this code :
+
+.worker/index.ts
+[source,typescript,linenums]
+----
+import { Module } from '@zetapush/core';
+import { StandardUserManagementModule, StandardUserWorkflow, ConfirmationUrlHttpHandler } from '@zetapush/user-management';
+
+@Module({
+  imports: [StandardUserManagementModule], <1>
+  expose: {
+    user: StandardUserWorkflow, <2>
+    http: ConfirmationUrlHttpHandler <3>
+  }
+})
+export default class Api {}
+----
+<1> First you need to export a class (named _Api_ here) that is a module (configured with the _@Module()_ annotation). You import the **StandardUserManagementModule** to use it into your application.
```

Comment:
You should put <1> on @Module line with this explanation: As usual, the worker code exports a class (maned _Api_here). As we import another module (`@zetapush/user-management`) we need to use @Module annotation.

Maybe we also need to make a reference/link to specific documentation about @Module.